### PR TITLE
Jira/flc 210 contract fabric handler

### PR DESCRIPTION
**Pull request by artemkanaki**

### CODE REVIEW

File: packages/fluence.lib.litecoin/src/services/blockchain/litecoin.ts [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+    public signDataAndStringify(privateKey: Buffer, data: Buffer): string {
+        const signature = this.signData(privateKey, data);
+        const hex = Buffer.alloc(64, Buffer.concat([ signature.r, signature.s ])).toString('hex');
+
+        return `0x${ hex }`;
+    }
+
+    private getBitcoinCoreNetwork() {
+        return this.getNetworkId() === AvailableNetwork.LITECOIN
```
Comment (andrew-zubko) [inline packages/fluence.lib.litecoin/src/services/blockchain/litecoin.ts]: not right

File: packages/fluence.lib.litecoin/src/services/blockchain/litecoin.ts [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+            ? bitcoin.networks.litecoin
+            : bitcoin.networks.testnet
```
Comment (bobchainer) [inline packages/fluence.lib.litecoin/src/services/blockchain/litecoin.ts]: not right

Comment (artemkanaki) [reply packages/fluence.lib.litecoin/src/services/blockchain/litecoin.ts]: can you please describe more specifically what exactly "not right"?

File: packages/fluence.lib.litecoin/src/services/blockchain/litecoin.ts [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+    public getServiceId() {
+        return `litecoin.${ this.getNetworkId() }.blockchain.service`;
+    }
+
+    public isValidNetwork(network: string) {
+        return AvailableNetwork.LITECOIN === network || AvailableNetwork.LITECOIN_TESTNET === network;
+    }
+
+    public async getHDAddress(index: number): Promise<string> {
+        const masterPubKey = config.has('multivest.blockchain.bitcoin.hd.masterPublicKey')
```
Comment (andapplctr) [inline packages/fluence.lib.litecoin/src/services/blockchain/litecoin.ts]: maybe litecoin?

**Review by andapplctr — APPROVED** [review]

**Pull request merged**